src/components/Arrays.jsx

```diff
@@ -0,0 +1,74 @@
+export const categoryArr = 
+[
+  { id: 1, category: "who said what"},
+  { id: 2, category: "prophecies, dreams, miracles"},
+  { id: 3, category: "minor prophets"},
+  { id: 4, category: "kings of judah/israel"},
+  { id: 5, category: "genealogy"},
+  { id: 6, category: "fill in the blank"}
+];
+
+export const dollarValArr = 
+[
+  { id: 1, dollarval: "$200"},
+  { id: 2, dollarval: "$400"},
+  { id: 3, dollarval: "$600"},
+  { id: 4, dollarval: "$800"},
+  { id: 5, dollarval: "$1000"}
+];
+
+export const tableArr =
+[  // 200 row
+  { id: 1, question: "'I am the light of the world.'", answer: "Jesus" },
```

Comment:
The way you broke out the category and dollar values look great!

Now you just need to tie those arrays back to the main tableArr...
```suggestion
{ 
  id: 1,
  question: 'I am the light...',
  answer: 'Jesus',
  categoryId: 1,
  dollarValId: 1,
}
```